decidim-core/app/controllers/decidim/geolocation_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Decidim
+  class GeolocationController < Decidim::ApplicationController
+    # overwrite original rescue_from to ensure we print messages from ajax methods (update)
+    rescue_from Decidim::ActionForbidden, with: :ajax_user_has_no_permission
```

Comment:
I see that is being used in several places now, can we create a reusable concern with this an the method `ajax_user_has_no_permission`. Then search for the other controllers and use the concern too.
Thanks!